.github/workflows/app-react.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - run: npm run build --if-present
         working-directory: ${{env.working-directory}}
 
-      - run: npm run coverage
+      - run: npm run coverage -- --maxWorkers=2
```

Comment:
this works because npm by default detects available processors and uses one less, to try and be less intrusive:
https://jestjs.io/docs/configuration#maxworkers-number--string
As a result this only uses one of our gha's two available cores.

Reply:
On my fork this cuts test execution time by 50%